.eslintrc

```diff
@@ -20,6 +37,27 @@
         "argsIgnorePattern": "^_",
         "ignoreRestSiblings": true
       }
-    ]
+    ],
+    /**
+     * This rules is set up to use the same rules as "@typescript-eslint/recommended" in v5.
+     * @see https://typescript-eslint.io/blog/announcing-typescript-eslint-v6/
+     */
```

Comment:
👋 typescript-eslint maintainer here - I just noticed this PR has been sitting for a little while. Would you like any help with it?